apitest/features/sqitch-nojwt.feature

```diff
@@ -15,3 +14,27 @@ Feature: Basic sqitch access control
         """
         And  I send a POST request to "http://{api_server}/comments"
         Then the response status should be "401"
+
+    Scenario: Get likert questions
+        When I send a GET request to "http://{api_server}/likert_questions"
+        Then the response status should be "200"
+        And the JSON response root should be array
```

Comment:
could you extend this validation a little bit to look inside the array and be sure we got a real question returned? so many of the responses are a json array i worry we could break the API by changing the json shape and not catch it with this test.

Reply:
if it's easier to test there is a query parameter to get a single object back that is not wrapped in an array